src/main/kotlin/no/nav/syfo/sykmeldingstatus/kafka/model/KafkaMetadata.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.syfo.sykmeldingstatus.kafka.model
+
+import java.time.LocalDateTime
+
+data class KafkaMetadata(
+    val sykmeldingId: String,
+    val timestamp: LocalDateTime
```

Comment:
Burde vi kanskje ha med en source, slik at vi vet hvor statusoppdateringen kommer fra, med tanke på at vi kanskje skal ha flere kilder etterhvert.

Reply:
Skal vi ha flere kilder..? Nå som vi ikke lenger setter sykmeldinger til utgått er jeg usikker på om det vil skje. På den annen side hender det at vi sletter sykmeldinger da.. Ja, kanskje source ikke er så dumt..?

Reply:
Jeg vet ikke, trodde det kanskje var noen muligheter for det, men det kommer vel kanskje ikke flere kilder egentlig.

Reply:
Man vet aldri da, så sånn sett er det kanskje bedre å ha source der :) Men det kan vi kanskje heller legge til som en del av https://trello.com/c/VmYz8GRO/950-skriv-statusendringer-til-nytt-topic ?